the second patch reverts the opus upgrade to v1.5.2
https://github.com/VCVRack/VCV-Recorder/commit/e2b6d883877316788f288d49f35c16003ab91334
as that version seem to have problems building properly on armv7l

diff --git a/Makefile b/Makefile
index 1781320..224c529 100644
--- a/Makefile
+++ b/Makefile
@@ -55,7 +55,7 @@ $(ffmpeg): $(lame) $(libopus)
 	# Don't use $(CONFIGURE) because this is a handwritten configure script
 	# ffmpeg bug: The pkgconfig dir is not set from pkgconfigdir at all. Set it with PKG_CONFIG_PATH instead.
 	# Use ./configure --help and --list-encoders and --list-muxers for reference
-	cd ffmpeg && PKG_CONFIG_PATH="$(DEP_PATH)/lib/pkgconfig" ./configure \
+	cd ffmpeg && PKG_CONFIG_PATH="$(DEP_PATH)/lib/pkgconfig" CPPFLAGS="-march=armv7 -mtune=cortex-a9 -mfpu=neon -mno-unaligned-access -DSIMDE_ENABLE_NATIVE_ALIASES" ./configure \
 		--prefix="$(DEP_PATH)" \
 		--extra-cflags="$(MAC_SDK_FLAGS)" \
 		--extra-ldflags="$(MAC_SDK_FLAGS)" \
@@ -82,11 +82,11 @@ $(lame):
 
 # libopus
 $(libopus):
-	$(WGET) -nc "https://downloads.xiph.org/releases/opus/opus-1.5.2.tar.gz"
-	$(SHA256) opus-1.5.2.tar.gz 65c1d2f78b9f2fb20082c38cbe47c951ad5839345876e46941612ee87f9a7ce1
-	cd dep && $(UNTAR) ../opus-1.5.2.tar.gz
-	cd dep/opus-1.5.2 && $(CONFIGURE)
-	cd dep/opus-1.5.2 && $(MAKE) install
+	$(WGET) -nc "https://archive.mozilla.org/pub/opus/opus-1.3.1.tar.gz"
+	$(SHA256) opus-1.3.1.tar.gz 65b58e1e25b2a114157014736a3d9dfeaad8d41be1c8179866f144a2fb44ff9d
+	cd dep && $(UNTAR) ../opus-1.3.1.tar.gz
+	cd dep/opus-1.3.1 && $(CONFIGURE)
+	cd dep/opus-1.3.1 && $(MAKE) install
 
 
 include $(RACK_DIR)/plugin.mk
